spec/requests/tasks_spec.rb

```diff
@@ -0,0 +1,51 @@
+describe 'TasksController' do
+  let(:task) { FactoryGirl.create(:miq_task, :state => MiqTask::STATE_FINISHED) }
+  let(:task2) { FactoryGirl.create(:miq_task, :state => MiqTask::STATE_FINISHED) }
+
+  def expect_deleted(*args)
+    args.each do |arg|
+      expect(MiqTask.find_by(:id => arg.id)).to be_nil
+    end
+  end
+
+  it 'deletes on DELETE' do
+    api_basic_authorize resource_action_identifier(:tasks, :delete, :delete)
+
+    url = api_task_url(nil, task.id)
+
+    delete(url)
+
+    expect(response).to have_http_status(204)
```

Comment:
For the http status, could you use `:forbidden` rather than `204`? will help to keep our specs consistent

Reply:
Probably not ... I guess `:empty`?

Reply:
Turns out `:no_content`, according to http://billpatrianakos.me/blog/2013/10/13/list-of-rails-status-code-symbols/

That's why I prefer the numerical values, no confusion about what it could be :).

Reply:
(fixed)